packages/core/src/routes/experience/classes/verifications/social-verification.ts

```diff
@@ -451,7 +457,7 @@ export class SocialVerification implements IdentifierVerificationRecord<Verifica
 
     const socialUserInfo = await this.libraries.socials.getUserInfo(
       this.connectorId,
-      connectorData,
+      transformConnectorData(connectorData),
```

Comment:
```suggestion
     normalizeConnectorData(connectorData),
```

So far we do not have a `transformXXX` naming convention yet, better use `normalize`

Reply:
nit:
Instead of wrapping all the logic inside the data formatting method, it's preferable to explicitly handle the validation logic at a higher level — for example:
`(isExternalGoogleOneTap ? parseGoogleOneTapConnectorData(connectorData) : connectorData)`

It is more readable and developer-friendly.